lib/krb5/keytab_any.c

```diff
@@ -222,7 +222,7 @@ any_remove_entry(krb5_context context,
 {
     struct any_data *a = id->data;
     krb5_error_code ret;
-    int found = 0;
+    size_t found = 0;
```

Comment:
'found' should be acceptable as an 'int'.   Its used as a boolean either zero or non-zero.

Given its usage setting `found = 1;` instead of `found++;` at line 229 is probably the correct fix.